### PR TITLE
[irods/irods2600] change copies to replicas in help text (4-3-stable)

### PR DESCRIPTION
**Pull request by trel**
cherry-pick of  #429

### CODE REVIEW

File: src/itrim.cpp [hunk 1]
```diff
@@ -122,17 +121,17 @@ usage() {
...
+        " -S  srcResource - specifies the resource of the replica(s) to be deleted.",
+        "     If specified, only replicas stored in this resource will be candidates",
         "     for the deletion.",
         " -v  verbose",
         " -V  Very verbose",
-        "--age age_in_minutes - The minimum age of the copy in minutes for trimming.",
-        "     i.e., a copy will not be trimmed if its age is less.",
-        "--dryrun - Do a dry run. No copy will actually be trimmed.",
+        "--age age_in_minutes - The minimum age of the replica in minutes for trimming.",
+        "     i.e., a repliaa will not be trimmed if its age is less.",
```
Comment (korydraughn) [inline src/itrim.cpp:133]: Dang. We missed one.

repliaa -> replica

Comment (korydraughn) [reply src/itrim.cpp:133]: Same thing in the PR that was just merged.

Comment (trel) [reply src/itrim.cpp:133]: noooo.    on it.

**Comment by trel** [comment]
done

**Review by korydraughn — APPROVED** [review]
Squash/Pound it.

**Comment by trel** [comment]
#'d

**Pull request merged**